packages/cli/src/init/init.js

```diff
@@ -73,35 +73,36 @@ function generateProject(destinationRoot, newProjectName, options) {
     yarn.getYarnVersionIfAvailable() &&
     yarn.isGlobalCliUsingYarn(destinationRoot);
```

Comment:
should this be handled by PackageManager as well?

Reply:
IMO yes. We should have one unit in the system that decides how to install packages and it should work the same way in every scenario. Usage is quite cool and if there will be another package manager in the future, we will just add it here :P

Reply:
Why it's not handled by PackageManager then? To be honest, I feel like it could be a great candidate for a separate package.

Reply:
It is used here to get `yarnVersion` and pass it to a function that actually initializes project (so much spaghetti code there). I wanted to change this as well, but it would require me to do a lot more changes, making this PR too big. I can create follow-up PR to use `PackageManager` everywhere.